app/assets/stylesheets/theme_previews.css

```diff
@@ -35,7 +35,7 @@ input[type='radio']:checked + label .theme-content {
   box-shadow: 0 2px 4px rgba(0,0,0,0.2);
 }
 
-@media (max-width: 1424px) {
+@media (max-width: 800px) {
```

Comment:
I updated this media query to stablize the resume view on smaller devices, before it was applying these styles to larger devices causing the issue with wider themes on ur desktop i.e it used 1424px as a dimension size and then sets the element to occupy 100% of the space available - causing the issue.